### PR TITLE
PLAT-723 Add Option to Use Variables in Data Import

### PR DESCRIPTION
**Pull request by thees-k**
### Test Plan

`EmfImportVariablesFinderTest` and `EmfImportVariablesReplacerTest`

#### How to try out:

Create a file with `.csv` suffix that has the following content:
```
$$a$$,127.0.0.1,$$b$$,username,secret-password,nix.com
test1,127.0.0.1,1234,$$c$$,secret-password,nix.com
$$a$$,127.0.0.1,1234,username,secret-password,$$d$$
``` 
Go to Page "Servers".

=> Click "IMPORT" and then "Select CSV File". Then upload the new created file.

The popup will show a table that contains the uploaded data:

=> Click "ENTER VARIABLE VALUES"

You will see text input fields for the defined variables:

=> Enter the variables values, then click "REPLACE VARIABLES"

The popup will show a table that contains the uploaded data with replaced variable definitions:

=> Click "ANALYZE"

If you have entered a value that cannot be parsed you will see a message:

=> Click "CLOSE"

=> Click "BACK"

=> Fix the wrong value, then click again "REPLACE VARIABLES"

=> Click again "ANALYZE"

The popup will show a preview of what will be imported:

=> Click "SAVE AND CLOSE"

You will see the imported data in the table:

### CODE REVIEW

File: platform-emf/src/main/java/com/softicar/platform/emf/EmfI18n.java [hunk 1]
```diff
@@ -118,6 +118,10 @@
 		.de("Aktiviert den Bearbeitungs-Modus für den gegebenen Eintrag.");
 	I18n0 ENTER_TARGET_INDEX = new I18n0("Enter target index")//
 		.de("Ziel-Index eingeben");
+	I18n0 ENTER_THE_VARIABLE_VALUES_FOR_THE_CURRENT_IMPORT = new I18n0("Enter the variable values for the current import.")//
+		.de("Geben Sie die Variablenwerte für den aktuellen Import ein.");
+	I18n0 ENTER_VARIABLE_VALUES = new I18n0("Enter variable values")//
```
Comment (oliver-richers) [inline platform-emf/src/main/java/com/softicar/platform/emf/EmfI18n.java:123]:
```suggestion
	I18n0 ENTER_VARIABLE_VALUES = new I18n0("Enter Variable Values")//
```

File: platform-emf/src/main/java/com/softicar/platform/emf/EmfI18n.java [hunk 2]
```diff
@@ -289,6 +299,8 @@
 		.de("Eintrag entfernen");
 	I18n0 REMOVE_FILE = new I18n0("Remove File")//
 		.de("Datei entfernen");
+	I18n0 REPLACE_VARIABLES = new I18n0("Replace variables")//
```
Comment (oliver-richers) [inline platform-emf/src/main/java/com/softicar/platform/emf/EmfI18n.java:302]:
```suggestion
	I18n0 REPLACE_VARIABLES = new I18n0("Replace Variables")//
```

File: platform-emf/src/main/java/com/softicar/platform/emf/EmfI18n.java [hunk 3]
```diff
@@ -420,4 +432,7 @@
 		.de("Falsche Spaltenanzahl.");
 	I18n0 YES = new I18n0("Yes")//
 		.de("Ja");
+	I18n2 YOU_CAN_DEFINE_VARIABLES_BY_ENCLOSE_THEM_WITH_DOUBLE_DOLLAR_SIGNS_FOR_EXAMPLE_ARG1_TO_DEFINE_VARIABLE_ARG2 = new I18n2(
+		"You can define variables by enclose them with double dollar signs, for example %s to define variable %s.")//
```
Comment (oliver-richers) [inline platform-emf/src/main/java/com/softicar/platform/emf/EmfI18n.java:436]:
```suggestion
		"You can define variables by enclosing them with double dollar signs, for example %s to define variable %s.")//
```

File: platform-emf/src/test/java/com/softicar/platform/emf/management/importing/variable/find/EmfImportVariablesFinderTest.java [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+import com.softicar.platform.common.container.map.set.SetMap;
+import com.softicar.platform.emf.management.importing.variable.EmfImportVariableCoordinates;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class EmfImportVariablesFinderTest extends Assert {
```
Comment (oliver-richers) [inline platform-emf/src/test/java/com/softicar/platform/emf/management/importing/variable/find/EmfImportVariablesFinderTest.java:12]: the indentation is this class looks strange

**Review by oliver-richers — CHANGES_REQUESTED** [review]
looks fine in general

File: platform-emf/src/test/java/com/softicar/platform/emf/management/importing/variable/find/EmfImportVariablesFinderTest.java (hunk 4 above)
Comment (thees-k) [reply platform-emf/src/test/java/com/softicar/platform/emf/management/importing/variable/find/EmfImportVariablesFinderTest.java:12]: re-formatted

**Review by oliver-richers — APPROVED** [review]

**Pull request merged**